### PR TITLE
Remove airflow-pr tool 

### PR DESCRIPTION
**Pull request by ryw**
Removes `airflow-pr` tool, which was linked to our former use of Jira, and this also removes LGPL (category-x) dependencies from /dev/requirements.txt which isn't a bad thing

re: https://github.com/apache/airflow/issues/10674

### CODE REVIEW

**Review by potiuk — APPROVED** [review]
LGTM. Just want to make 100% sure that we are not using it any more @kaxil ?

File: dev/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,4 @@
 click~=7.0
-jira>=2.0.0
 keyring==10.1
 gitpython
 jinja2~=2.10
-PyGithub
```
Comment (kaxil) [inline dev/requirements.txt:6]: PyGithub is used here: https://github.com/apache/airflow/blob/master/dev/airflow-github#L28

File: dev/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,4 @@
 click~=7.0
-jira>=2.0.0
 keyring==10.1
 gitpython
```
Comment (kaxil) [inline dev/requirements.txt]: You can remove `gitpython` though

File: dev/requirements.txt (hunk 1 above)
Comment (ryw) [reply dev/requirements.txt:6]: ok - added it back in

**Review by kaxil — APPROVED** [review]

**Comment by potiuk** [comment]
🎉  - 1117 lines of code. I love those commits. They are the best ones. 🎉

**Comment by ryw** [comment]
thanks @potiuk :)

**Pull request merged**